src/clarity/wizard/wizard.spec.ts

```diff
@@ -466,6 +480,15 @@ describe("Wizard", () => {
             expect(tab3).toBeDefined();
             expect(tab4).toBeDefined();
         });
+
+        it("calls the user-defined onCancel handler when the Cancel button is clicked", () => {
+            fixture.detectChanges();
+            doCancel(compiled);
+
+            expect(fixture.componentInstance.hasBeenCanceled).toBe(true);
+            spyOn(instance, "_close");
```

Comment:
i changed the test to listen to the wizard instance .onCancel method but it doesn't detect it.
If i check the component instance the test succeed : 

`

     const myComponentInstance = fixture.componentInstance;
     spyOn(myComponentInstance, "myOnCancel");
     doCancel(compiled);
     expect(myComponentInstance.myOnCancel).toHaveBeenCalled();
`

How could i modify the current test to make it works ?